Dockerfile.builder

```diff
@@ -14,7 +14,7 @@ WORKDIR /go/src/github.com/cilium/cilium
 ENV GOROOT /usr/local/go
 ENV GOPATH /go
 ENV PATH "$GOROOT/bin:$GOPATH/bin:$PATH"
-ENV GO_VERSION 1.12.8
+ENV GO_VERSION 1.13
```

Comment:
Is there a reason this is 1.13 vs. 1.13.0?

Reply:
I'll see you in the 1.14 release

https://github.com/cilium/cilium/pull/7211#discussion_r261011298